pkgs/development/python-modules/lexilang/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "LexiLang";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-QAKJZRc29GssiLX2q9oRr5SYfaTDDrlk7EELiMxWiSU";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  doCheck = false;
```

Comment:
Please add a comment above doCheck, explaining why it's disabled?

Reply:
I couldn't find out reason for error:
```
python3.11-LexiLang> Running phase: pytestCheckPhase
python3.11-LexiLang> Executing pytestCheckPhase
python3.11-LexiLang> ============================= test session starts ==============================
python3.11-LexiLang> platform linux -- Python 3.11.7, pytest-7.4.3, pluggy-1.3.0
python3.11-LexiLang> rootdir: /build/LexiLang-1.0.1
python3.11-LexiLang> collected 0 items
python3.11-LexiLang> ============================ no tests ran in 0.01s =============================
python3.11-LexiLang> /nix/store/v5irq7wvkr7kih0hhnch5nnv2dcq8c4f-stdenv-linux/setup: line 1557: pop_var_context: head of shell_variables not a function context
```

Reply:
Collected 0 items means it didn't the tests. Check if they are in the distribution you're pulling. If not, please fetch from git.

Reply:
It is possible that these files are required - https://github.com/LibreTranslate/LexiLang/tree/main/dictionaries
I just don't know how to download them separately yet.

Reply:
https://github.com/LibreTranslate/LexiLang/blob/main/test.py

I think this is the only test. I'm not sure about the dictionary files linked, but they're also not part of the sdist.

```nix
checkPhase = ''
  runHook preCheck
  ${python.interpreter} test.py
  runHook postCheck
'';

Reply:
Thanks, fixed!